src/services/dealer.ts

```diff
@@ -225,3 +225,31 @@ export const setLogo = async ({
     return handleValidationError(error, { swallowErrors: true })
   }
 }
+
+export const setImage = async ({
+  dealerId,
+  image,
+  options = {},
+}: {
+  dealerId: number
+  image: string
```

Comment:
Can we call it `imageUrl` or `imageS3Key`? When I read image here I expect an image object.

Reply:
keep it in sync with the other places where we upload images :)

Reply:
what do you mean? Lets keep `image` since this is the field name in BE

Reply:
yeah, I guess that's what I mean... I'd value consistency with other api calls and backends naming over clarity